values.yaml

```diff
@@ -133,6 +133,9 @@ ui:
   service:
     enabled: true
     type: null
+    # Define annotations to utilize AWS Load balancers according to https://docs.aws.amazon.com/eks/latest/userguide/load-balancing.html
+    # annotations:
+        # service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
```

Comment:
This just seems generally useful for _any_ sort of annotation. Does it make sense to call out ELB-related concerns specifically?

Reply:
You're right, thanks for the feedback.  I've updated the comments and removed aws specific